backend/src/main/java/de/bund/digitalservice/ris/adm_vwv/application/converter/transform/DateToQuoteTransformer.java

```diff
@@ -17,7 +17,8 @@ public class DateToQuoteTransformer {
   /**
    * Transforms the {@code AkomaNtoso} object to a date to quote string.
    *
-   * @return The (first) date to quote, or {@code null} if the surrounding {@code <proprietary>}
+   * @return The (first) date to quote, or {@code null} if the surrounding
+   *         {@code <proprietary>}
    *         or {@code <dateToQuoteList>} elements are {@code null}
    */
   public String transform() {
```

Comment:
Not that much important here, but a typical style in Java is to mark something deprecated and use the newer implementation such as:
```suggestion
/**
 * @deprecated Use #transformList() instead
 */
public String transform() {
  var dateToQuoteList = transformList();
  return dateToQuoteList.isEmpty() ? null : dateToQuoteList.getFirst();
}
```

Reply:
Nice, love it!